packages/zosfiles/src/methods/utilities/Utilities.ts

```diff
@@ -20,6 +20,8 @@ import * as path from "path";
 import { ZosFilesUtils } from "../../utils/ZosFilesUtils";
 
 export class Utilities {
+    private static readonly minimumTimeout = 5;
```

Comment:
should this timeout be added as a constant? in Constants.ts? only because a min timeout of 5 is used a lot of places

Reply:
I only did this here because of the several uses in this one file. The only other locations where I could find `5` being used was in the `zosfiles` strings file, and as a parameter for responseTimeout in the `zosmf` package. The only places where the change could be placed to share that value across those three files is either `zosmf` or `core`, and a new constants object would need to be created to export it. Seeing as the property here is readonly, I plan to leave the implementation as-is until we find more constants that we would like to expose at a lower level.